### PR TITLE
Issue122 move niceday client to own repo

### LINKED ISSUE #122: Move niceday_client to its own repo
niceday_client is used by `Rasa_Bot` and onboarding script, to make the docker installs cleaner and make things more modular we will move it out of this monolith.

AB#2050

### PR DESCRIPTION
**Pull request by raar1**
Fixes #122 

`niceday_client` is now in its own repo and is installable as a python package:
<https://github.com/PerfectFit-project/niceday_client>

I updated the `onboarding` script (very simple) and it works. I also updated the only reference I found in `Rasa_Bot` and I think it's ok but maybe Sven or Nele can test this to make sure it's still running ok.

### CODE REVIEW

**Comment by nelealbers** [comment]
@raar1 The reference in Rasa_Bot is from the custom channel that Sven has been working on. I do not know whether that is done yet and how it is supposed to work, so Sven would be the better person to test this. Also I have never been able to get the onboarding script to work (kept getting some context cancelled error that probably has something to do with me using Windows again, but I have not yet gotten around to fix that), so Sven would also be the better person to test the update to the onboarding script. My apologies I can't help right now!

**Comment by raar1** [comment]
@nelealbers No problem at all, Nele! I'll grab Sven on Monday :)

File: Rasa_Bot/requirements.txt [hunk 1]
```diff
@@ -2,4 +2,4 @@
 rasa==2.8.1 # NB! when updating, make sure to also update:
             # * rasa-sdk base image in Dockerfile, 
             # * docker-compose files (both in Rasa_Bot and root),
-            # * format of stories, domain.yml etc. if necessary
```
Comment (svenvanderburg) [inline Rasa_Bot/requirements.txt:5]: You accidentally removed a line here

**Review by svenvanderburg — APPROVED** [review]
Looks good! I added one very minor comment and it seems the CI is still breaking. The most important thing is this though:

We now have an actual requirement for rasa server. This is slightly problematic, because for the rasa server (i.e. not the action server) we dependent on the `rasa/rasa:2.8.1-full` docker image. So using the custom channel will break if we run the application with docker compose. The custom channel is still disabled though, so this will be fixed with #21 (I added a comment to that issue referring to this PR).

**Pull request merged**